pom.xml

```diff
@@ -120,7 +120,7 @@
         <yarn.version>v1.5.1</yarn.version>
 
         <!--Project properties-->
-        <ddf.version>2.26.1</ddf.version>
+        <ddf.version>2.26.18-SNAPSHOT</ddf.version>
```

Comment:
Is it correct to be using a DDF snapshot version dependency as opposed to a released version?

Reply:
I think it's ok as long as it's temporary. These changes depend on changes in that snapshot version, so as long as we update when that version is released officially, we should be okay